cocos/2d/CCActionInterval.cpp

```diff
@@ -298,6 +314,18 @@ Sequence::~Sequence(void)
 
 void Sequence::startWithTarget(Node *target)
 {
+    if (target == nullptr) {
+        log("Sequence::startWithTarget error: target is nullptr!");
+        return;
+    }
+    if (_actions[0] == nullptr || _actions[1] == nullptr) {
+        log("Sequence::startWithTarget error: _actions[0] or _actions[1] is nullptr!");
+        return;
+    }
+    if (_duration <= FLT_EPSILON) {
+        return;
+    }
```

Comment:
I have two questions:

* Can `_duration` be 0? 
* `FLT_EPSILON` is a very small value, it is used to check if two floats are equal, so i think it is not a good to use it here